articles/tutorials/building_2d_games/04_working_with_textures/index.md

```diff
@@ -0,0 +1,344 @@
+| ---------- | ----------- | ----------------------------------------------------------------------------------------------------------------- |
+| *texture*  | [Texture2D](xref:Microsoft.Xna.Framework.Graphics.Texture2D) | The [Texture2D](xref:Microsoft.Xna.Framework.Graphics.Texture2D) to draw                                                                                           |
+| *position* | [Vector2](xref:Microsoft.Xna.Framework.Vector2)   | The xy-coordinate position to draw the texture at. The origin point being the top-left corner of the image.       |
+| *color*    | [Color](xref:Microsoft.Xna.Framework.Color)     | The color mask (tint) to apply to the image drawn. Specifying [Color.White](xref:Microsoft.Xna.Framework.Color.White) will render the texture with no tint. |
+
+Try adjusting the position and color parameters to see how they can affect the image being drawn.
+
+In MonoGame, the xy-coordinate space on the screen begins at (0, 0) in the top left corner, with the x-coordinate increasing to the right, and the y-coordinate increasing downward. Knowing this, let's try drawing the logo centered on the game window. To do this, we would first need to know the xy-coordinate position of the center of the game window. We can get this information from the [Window.ClientBounds](xref:Microsoft.Xna.Framework.GameWindow.ClientBounds) property provided by the [Game](xref:Microsoft.Xna.Framework.Game) class.
+
+[Window.ClientBounds](xref:Microsoft.Xna.Framework.GameWindow.ClientBounds) has both an `X` and `Y` property that represent the top-left coordinates of the game window (0, 0). It also `Width` and `Height` properties that represent the width and height of the game window in pixels. Knowing this, we only need to get half of the width and height to know the xy-coordinate position of the center of the game window. Let's update our draw method to use this information, adjust the [Draw](xref:Microsoft.Xna.Framework.Game.Draw(Microsoft.Xna.Framework.GameTime)) method call to the following
+
+```cs
+_spriteBatch.Draw(_logo, new Vector2(Window.ClientBounds.Width, Window.ClientBounds.Height) * 0.5f, Color.White);
+```
+
+> [!TIP]  
+> In the example above, we multiply the [Vector2](xref:Microsoft.Xna.Framework.Vector2) created by `0.5f` to half the value instead of dividing it by `2.0f`. If you're not used to seeing this, it might seem strange at first, but it's actually an optimization technique. CPUs are able to perform multiplication operations much faster than division operations and reading `* 0.5f` is easily understood to be the same thing as `/ 2.0f` when reading.
+
+We're now telling the position to be a half the width and height of the client bounds. If we run the game now, the logo should be drawn at the center of the game window correct? Well, not exactly. If you run the game now, it will look similar to the following:

```

Comment:
I do like this step by step, try, fail, try fail and understand why at each phase.


P.S. ----- "We are" not "We're"  :D

Reply:
We're corrected to We are